assets/src/components/v2/on_bus/no_service.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { ComponentType } from "react";
+import Header from "../departures/header";
```

Comment:
Normally when importing something from outside a component's directory we'll use the `Components` alias, which maps to the root `components` directory. This way, reorganizing files/directories is less likely to require noisy changes to the number of `..`s in an import path.

```ts
import Header from "Components/v2/departures/header";
```